.buildkite/scripts/steps/functional/apm_cypress.sh

```diff
@@ -29,8 +29,15 @@ fi
 
 echo "--- APM Cypress Tests"
 
-cd "$XPACK_DIR"
+pushd "$XPACK_DIR"
 
-node solutions/observability/plugins/apm/scripts/test/e2e.js \
+if ! node solutions/observability/plugins/apm/scripts/test/e2e.js \
   --kibana-install-dir "$KIBANA_BUILD_LOCATION" \
-  $CYPRESS_ARGS
+  $CYPRESS_ARGS ; then
+
+  # Report the error to the team using the subfolder matching the Slack Team
+  popd
+  mkdir "$REPORT_SLACK_TEAM"
+  touch "$REPORT_SLACK_TEAM"/obs-ux-infra_services-team.slack
```

Comment:
Can you help me understand the changes here vs passing a string `PING_SLACK_TEAM='obs-ux-infra_services-team`?

I'm not opposed to files but it seems to add some additional complexity.

Reply:
Sure thing.

I wanted to make [pingTeam](https://github.com/elastic/kibana/pull/205260/files#diff-3ca95c2f7d9589fd2954d0691e1ca0a4c2666b5ee8c6c2583242a5b97b5b253bR192) agnostic to the team name. But read the files under `target/report-slack-team`, and those files match the name of the slack team to be notified if any errors.

Regarding your question about `PING_SLACK_TEAM='obs-ux-infra_services-team'`, I'm not sure how the `post_command` can read that value and whether I could use something like `ts-node .buildkite/scripts/lifecycle/ping_team_ftrs.ts "$PING_SLACK_TEAM"` in [here](https://github.com/elastic/kibana/pull/205260/files#diff-77757ca90ebd29aff382099edd9552dd72134c1bb5ca5a2dedfe0443e0709beaR57)

Maybe I could use the `buildkite metadata set` command to notify the Slack team to reduce the additional complexity with files under `target/report-slack-team`.